apps/src/gamelab/animationListModule.js

```diff
@@ -164,6 +164,54 @@ function animationPropsReducer(state, action) {
 }
 
 /**
+ * Used to order numbers for sort()
+ */
+function compareNumbers(a, b) {
+  return a - b;
+}
+
+/**
+ * Given a name and an animationList, determine if the name is unique.
+ */
+export function isNameUnique(name, animationList) {
+  for (let animation in animationList) {
+    if (animationList[animation].name === name) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
Oo this is cool. Unfortunately animationList isn't an array here (only `orderedKeys` is an array). I'll add a comment to clarify.